testdata/datetimes.txt

```diff
@@ -0,0 +1,50 @@
+openapi-httprequest api.yaml
```

Comment:
It would be great to have a comment at the top of each that summarises what the
test is testing without the need to look through the actual YAML too hard.

The name of this test file is "datetimes.txt" but I don't see dates and times anywhere.